client/src/dashboard/RewritingRuleModal.js

```diff
@@ -182,11 +183,37 @@ const RewritingRuleModal = NiceModal.create(({user_id, rule=null, query=null}) =
     });
   };
 
-  function findDistanceToSpaces(inputString, currentPosition) {
+  const onBeautifyQuery = () => {
+    const sqlLanguage = (queryLanguage === 'pgsql') ? "postgresql" : queryLanguage;
```

Comment:
Can you clarify what this does?

Reply:
Do you mean function onBeautifyQuery? It is the function which uses sql-formatter library to re-format q0 and q1(beautify q0 and q1).